docs/data/pha.rst

```diff
@@ -373,6 +373,47 @@ filtered range to the requested range becomes larger::
   >>> print(pha3.get_filter())
   0.525600001216:10.906200170517
 
+Manipulating data
+=================
+
+Methods like :py:meth:`~sherpa.astro.data.DataPHA.get_dep` will
+apply the necessary grouping and filters, but it can be useful to
+convert other arrays, which can be done with
```

Comment:
How about:
>>> Methods like :py:meth:`~sherpa.astro.data.DataPHA.get_dep` will
apply the necessary grouping and filters to the PHAdata itself, but it can be useful to
convert other arrays, such as the counts array, which can be done with

Reply:
I've re-worded it slightly but not quite like you suggest.